### ISSUE TITLE
Add support for :key-fn in wrap-json-body middleware.

### ISSUE THREAD

**Issue by benjamincassidy** [ring-clojure/ring-json#51]
Add support for a :key-fn to the wrap-json-body middleware which is passed to cheshire to convert keys.

**Comment by redwoolf** [comment]
fixes #51

**Comment by bigwheels16** [comment]
Although it's not documented, you can pass a function as the :keywords? value:

```(def app (-> routes
                      app-routes
                     (wrap-json-body {:keywords? #(keyword (.replace % \_ \-))})))```
